doc/source/building/index.rst

```diff
@@ -352,17 +352,20 @@ virtual environments:
     Then install the Python-level dependencies (see ``pyproject.toml``) from
     PyPI with::
 
-       # Build dependencies
-       python -m pip install numpy cython pythran pybind11 meson-python ninja pydevtool rich-click
+       # All dependencies
+       python -m pip install -r requirements/all.txt
 
-       # Test and optional runtime dependencies
-       python -m pip install pytest pytest-xdist pytest-timeout pooch threadpoolctl asv gmpy2 mpmath hypothesis
+       # Alternatively, you can install just the dependencies for certain
+       # development tasks:
 
-       # Doc build dependencies
-       python -m pip install sphinx "pydata-sphinx-theme==0.9.0" sphinx-design matplotlib numpydoc jupytext myst-nb
+       # Build and dev dependencies (for `python dev.py {build, lint, mypy}`)
+       python -m pip install -r requirements/build.txt -r requirements/dev.txt
```

Comment:
Unless we want to split out the extra dev dependencies for typing and linting into a separate file like `requirements/dev-extra.txt`?

Reply:
It seems fine as is.